packages/medusa/package.json

```
@@ -45,7 +45,7 @@
     "@inquirer/checkbox": "^2.3.11",
     "@medusajs/admin-sdk": "0.0.1",
     "@medusajs/core-flows": "^0.0.9",
-    "@medusajs/framework": "^0.0.1-preview-20240724081425",
+    "@medusajs/framework": "workspace:^",
```

Only apply this change before merging

```suggestion
    "@medusajs/framework": "^0.0.1-preview-20240724081425",
```